### DISCUSSION TITLE
Utracone hasło Global Administratora w Azure AAD

### DISCUSSION THREAD

**Discussion by nightdancerpl** [AzureCommunityPL/group#375] [Pomoc]
Witam, utraciłem hasło do Global Administratora w usłudze Azure AAD, w jaki sposób mogę je odzyskać. Nie ma żadnego innego administratora oraz nie miałem ustawionych metod odzyskiwania.

**Comment by eklime** [comment]
https://docs.microsoft.com/en-us/office365/troubleshoot/sign-in/forgot-sign-in-password

**Reply by nightdancerpl** [reply]
> https://docs.microsoft.com/en-us/office365/troubleshoot/sign-in/forgot-sign-in-password

Ehh
-Nie mam włączonego resetowania haseł
-Jest to jedyne konto administratora, nie ma innego

**Reply by TomaszOledzki** [reply]
Pozostaje Ci kontakt z supportem + dużo cierpliwości w tłumaczeniu dlaczego sam nie możesz odzyskać hasła i udowadnianiu, że to Twój tenant.
